.azure-pipelines/default.yml

```diff
@@ -0,0 +1,28 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+     
+     mkdir dist
+    displayName: 'Prep Environment, Install Packages'
+
+  - script: 'python ./build_package.py --dest %BUILD_SOURCESDIRECTORY%\dist "azure-keyvault"'
+    displayName: 'Generate KeyVault Tarball'
```

Comment:
How much stuff gets put into this dest directory? We should try and only publish the artifacts we need to persist and intermediate files as well.

Reply:
Only the build artifacts get dropped. Specifically, 1 wheel + 1 zip per `build_package` call.